app/forms/questionnaire_form.py

```diff
@@ -310,20 +310,25 @@ def _get_period_range_for_single_date(
         date_to: Mapping[str, dict],
     ) -> timedelta:
         list_item_id = self.location.list_item_id if self.location else None
-        value_source_resolver = ValueSourceResolver(
+        rule_evaluator = RuleEvaluator(
```

Comment:
I'm not sure this makes sense at the moment, the rule evaluator now has a value source resolver but you're still passing both to the field handler?

Reply:
Removed VSR 👍